.github/workflows/publish.yml

```diff
@@ -1,11 +1,16 @@
 name: "Publish"
 on:
-  push:
+  pull_request:
```

Comment:
I think this should be on: tag and branches: master; hotfix/**

Reply:
I think it's better to trigger when a PR is closed because any push for master or hotfix/** will trigger the workflow.